services/tracer/TracingEchoInstantiation.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef SERVICES_TRACING_ECHO_INSTANTIATIONS
+#define SERVICES_TRACING_ECHO_INSTANTIATIONS
+
+#include "hal/interfaces/SerialCommunication.hpp"
+#include "services/tracer/Tracer.hpp"
+#include "services/util/EchoInstantiation.hpp"
+#include "services/util/SesameCobs.hpp"
+#include "services/util/SesameWindowed.hpp"
+#include "services/util/TracingEchoOnSesame.hpp"
+
+namespace main_
+{
+    template<std::size_t MessageSize>
+    struct TracingEchoOnSesame
+    {
+        TracingEchoOnSesame(hal::BufferedSerialCommunication& serialCommunication, services::MethodSerializerFactory& serializerFactory, services::Tracer& tracer)
+            : cobs(serialCommunication)
```

Comment:
Consider renaming this to sesameCobs and sesameWindowed instead, can be misleading at first glance

Reply:
I thought smaller names are fine here